addons/hr_expense/static/src/views/attachment_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.AttachmentView" t-inherit-mode="extension" owl="1">
+       <xpath expr="//h3" position="replace">
+           <h3 t-if="!state.thread.mainAttachment.isPdf" class="mt0 mb8 ps-2 text-muted d-flex justify-content-center">
+               <div>
+                   <span t-if="['hr.expense', 'hr.expense.sheet'].includes(state.thread.model)">
+                       <t t-esc="state.thread.name" t-if="state.thread.model=='hr.expense'"/>
+                       <t t-esc="state.thread.mainAttachment.res_name or state.thread.name" t-if="state.thread.model=='hr.expense.sheet'"/>
```

Comment:
```suggestion
                       <t t-esc="state.thread.name" t-if="state.thread.model === 'hr.expense'"/>
                       <t t-esc="state.thread.mainAttachment.res_name or state.thread.name" t-if="state.thread.model === 'hr.expense.sheet'"/>
```
Triple === to ensure strict equality.

Also I would put the `t-if` before `t-esc`, but that's debatable :)

Reply:
Oh also, shouldn't we use `t-out` now instead of `t-esc` ? Not a 100% sure for JS templates

Reply:
They are using t-esc in the original template so i just copy paste it, don't know if there is a preference between one or the other i will check for that thx !

Reply:
I've asked and it seems that it's not really an error but we have to use t-out so i will change that :smile: